.github/workflows/ci.yml

```diff
@@ -37,22 +47,24 @@ jobs:
           rm -fr "${raw_cov_dir}"
           mkdir -p "${raw_cov_dir}"
 
+          # Print executed commands to ease debugging
+          set -x
```

Comment:
I'm still against keeping `set -x` by default. It's something that we need to debug the action, sure, but it should be enabled only when debugging is needed.

Reply:
Some failures are hard to reproduce, so it's not as easy as adding `set -x` and re-running the job again.  The default output of the CI job should be at a balance between too much output, which makes it hard to find the relevant output, and enough output to allow us debugging failures. I added the `set -x` just before the commands for which it might not be clear where the output is coming from, i.e. the `go test` and `go tool` commands.